app/songflong/__init__.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import uuid
+from pathlib import Path
+from queue import Queue
+from threading import Thread
+from time import time
+
+from .ffmpeg import ffmpeg_merge_video_audio
+
+from .download import download_video_stream, download_audio_stream
+from .links import get_youtube_link, find_all_links
+from .search import get_songs_by_bpm, get_track_bpm
+
+logging.basicConfig(level=logging.INFO,
+                    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+logger = logging.getLogger(__name__)
```

Comment:
Shouldn't initializing the logger be in app.py?

Reply:
doesn't matter